opencti-platform/opencti-front/src/private/components/workspaces/investigations/InvestigationGraph.jsx

```diff
@@ -2012,9 +2081,17 @@ class InvestigationGraphComponent extends Component {
                   onReset={this.onResetExpandElements.bind(this)}
                 />
               </Dialog>
+
+              <InvestigationRollBackExpandDialog
+                isRollBackPreExpandStateDialogOpen={isRollBackPreExpandStateDialogOpen}
```

Comment:
I would remain this prop to simple "open" as in the context of the dialog, it's clear enough.
When I'm naming a prop, I just think "in the context of the component"  and not relative to how it will be used by the parent(s)
The state name though can be more precise of course.